### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by feefladder**
add description about injection-safeness of query-with-bindings

### CODE REVIEW

**Comment by feefladder** [comment]
#118

File: README.md [hunk 1]
```diff
@@ -266,6 +266,21 @@ There are a couple of things you can do before panicking, namely:
...
 
+- SQLite's `query_with_bindings` as also used by `update_rows` is injection-safe. That is, any attempt to use sql inside a bound variable will escape it and insert it directly into the record. So the two equivalent statements:
+
+  ```gdscript
+  var table_name := "characters"
+  db.query_with_bindings("UPDATE "+ table_name +" SET level=? WHERE id=?;", ["level+1", 1])
+  db.update_rows(table_name, "id=1", {"level":"level+1"})
+  ```
+  
+  will insert a literal `'level+1'` into the database, in stead of incrementing the value by one. In stead, build a direct query like before:
```
Comment (2shady4u) [inline README.md:277]: instead is one word

File: README.md [hunk 2]
```diff
@@ -266,6 +266,21 @@ There are a couple of things you can do before panicking, namely:
     db.query_with_bindings("UPDATE "+ table_name +" SET "+ column_name +"=? WHERE id=?;", [100, 1])
     ```
 
+- SQLite's `query_with_bindings` as also used by `update_rows` is injection-safe. That is, any attempt to use sql inside a bound variable will escape it and insert it directly into the record. So the two equivalent statements:
```
Comment (2shady4u) [inline README.md:269]: SQLite's `query_with_bindings`, as also used internally by `update_rows`, is injection-safe.

Comment (2shady4u) [inline README.md:269]: inside **of** a bound variable
will escape ~~it~~ and insert it directly

File: README.md (hunk 1 above)
Comment (2shady4u) [inline README.md:277]: What does "like before" mean in this context? Which "before" are we talking about?

**Review by 2shady4u — CHANGES_REQUESTED** [review]
Hi @feefladder !

Sorry for the long time it took to review this PR.
I have requested some small syntax changes 😄

File: README.md (hunk 1 above)
Comment (feefladder) [reply README.md]: I thought it referred to the query-with-bound-column-name example, but that still used `query_with_bindings` in the solution..

**Comment by feefladder** [comment]
thanks for nitpicking!

**Review by 2shady4u — APPROVED** [review]

**Comment by 2shady4u** [comment]
Hi @feefladder 

Thank you for the update and apologies for the >2weeks ago it took for me to get to this 😄

**Pull request merged**